2017-02/feb-22.md

```diff
@@ -27,14 +30,21 @@ We've added a few folks as collaborators along the way!
 | github | [@aaronang](https://github.com/aaronang) | [@yavorsky](https://github.com/yavorsky) | [@STRML](https://github.com/STRML) | [@chicoxyzzy](https://github.com/chicoxyzzy) |
 | twitter | [@_aaronang](https://twitter.com/_aaronang) | [@yavorsky_](https://twitter.com/yavorsky_) | [@STRML_](https://twitter.com/STRML_) | [@chicoxyzzy](https://twitter.com/chicoxyzzy) |
 
-- Aaron just joined our slack this past week and as been super excited about getting involved: made a few PRs and efforts to improve our site and contributing docs.
+- Aaron just joined our slack this *past week* and has been super excited about getting involved: made a few PRs and efforts to improve our site and contributing docs.
 - Artem has been making a lot of great PRs to babel-preset-env and is also getting started with other stuff.
 - Samuel has made most of the PRs for fixing the bugs that we've having with the react constant elements plugin as well as added discourse comments support to our docs pages.
 - Sergey helps maintain compat-table and has been helping with babel-preset-env.
 
 ### Future Discussions
 
-> TODO
+> babel-preset-env and its future
+
+Should preset-env be in core? How can it be better integrated with babili/webpack?
```

Comment:
I think it can still be that tool for code transformation with env.

Env can act as the compile target. The advantage of env in core is that now every plugin can use this feature to detect the compile target the user intends and then provide the necessary transformations based on that. For example, you can understand the target Language and create arrowFns instead of plain functions - and not depend on another transformation step like es2015/env again to convert plainFns to arrows or vice versa.